Sources/ETDistribution.swift

```diff
@@ -125,6 +143,9 @@ public final class ETDistribution: NSObject {
     }
     var request = URLRequest(url: url)
     request.httpMethod = "GET"
+    if let accessToken = accessToken {
+      request.setValue("Bearer \(accessToken)", forHTTPHeaderField: "Authorization")
```

Comment:
In the response handling I think we need a case for when the authorization was invalid. Like if the token was manually revoke. Seems like right now the user will be stuck until the system time > refresh time. Instead if the authorization is invalid I think we should try to refresh it and if that fails show the login screen. We can also handle the case where the `loginSetting` wasn't provided but we get back an authorization error to automatically show the login screen